docs/core/migration/20-21.md

```diff
@@ -25,6 +25,18 @@ For an overview of the new features in .NET Core 2.1, see [What's new in .NET Co
   * [dotnet-user-secrets](https://github.com/aspnet/DotNetTools/blob/dev/src/dotnet-user-secrets/README.md) (Microsoft.Extensions.SecretManager.Tools)
   * [dotnet-sql-cache](https://github.com/aspnet/DotNetTools/blob/dev/src/dotnet-sql-cache/README.md) (Microsoft.Extensions.Caching.SqlConfig.Tools)
   * [dotnet-ef](/ef/core/miscellaneous/cli/dotnet) (Microsoft.EntityFrameworkCore.Tools.DotNet)
+  
+  In previous .NET Core SDK versions, the reference to one of these tools in your project file looks similar to the following example:
+
+  ```xml
+  <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="2.0.0" />
+  ```
+
+  Since this entry isn't used by the the .NET Core SDK any longer, you'll see a warning similar to the following if you still have references to one of these bundled tools in your project:
+  
+  `The tool 'Microsoft.EntityFrameworkCore.Tools.DotNet' is now included in the .NET Core SDK. Here is information on resolving this warning.`
+  
+  Removing the `<DotNetCliToolReference>` reference from your project file fixes this issue.
```

Comment:
nit: instead of "refererence", should it be "element"?

Reply:
see if it makes more sense now @BillWagner